spec/unit/workload_mapper_spec.rb

```diff
@@ -212,6 +212,15 @@
         }
       }
     }
+
+    let(:unlinked_services_one_missing) {
+      {
+       "web" => {
+          "links" => ["db"]
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.